Tools/scripts/build_binaries.sh

```diff
@@ -358,21 +314,23 @@ build_arducopter() {
         echo "Building ArduCopter $tag PX4$framesuffix binaries"
         ddir="$binaries/Copter/$hdate/PX4$framesuffix"
         skip_build $tag $ddir && continue
-        for v in v1 v2 v3 v4; do
-            make px4-clean
-            make px4-$v$framesuffix -j2 || {
-                echo "Failed build of ArduCopter PX4 $tag for $v"
+        for v in v1 v2 v3 v4 v4pro; do
+            skip_board_waf px4-$v && continue
+            waf configure --board $b --out $BUILDROOT clean && \
+                waf build --target bin/arducopter$framesuffix || {
+                echo "Failed build of ArduCopter$framesuffix PX4 $tag for $v"
+                error_count=$((error_count+1))
+                continue
+            }
+            cp -f $BUILDROOT/px4-$v/bin/arducopter"$framesuffix".px4 ArduCopter-$v.px4 || {
+                echo "Failed build copy of ArduCopter$framesuffix PX4 $tag for $v"
                 error_count=$((error_count+1))
                 continue
             }
+            copyit ArduCopter-$v.px4 $ddir $tag
```

Comment:
why do we keep this in a separate loop from the Linux boards?  aerofc-v1 is built there together with the Linux boards and add a .px4 suffix where needed.